documentation/develop/web-ext-command-reference.md

```diff
@@ -560,3 +568,4 @@ WEB_EXT_VERBOSE=true
 {%- include up-next.html -%}
 
 <!-- END: Up Next -->
+
```

Comment:
Nit, the newline added to the end of the file is likely not needed, but not a big deal.

Reply:
Sorry, auto-added by my editor. Maybe a case for [`.editorconfig`](https://editorconfig.org/)? It has a config for that.